app/pages/lab/workflow.cjsx

```diff
@@ -437,8 +441,18 @@ EditWorkflowPage = createReactClass
 
             </div>}
 
-          <div>
-            <a href={@workflowLink()} className="standard-button" target="from-lab" onClick={@handleViewClick}>Test this workflow</a>
+          <div className={if @isThereNotADefinedTask() then 'disabled-section' else ''}>
+            <a
+              disabled={@isThereNotADefinedTask()}
```

Comment:
Links can't be disabled by adding a `disabled` attribute. Render the link's content as plain text if you don't want it to be clickable, or just don't render the link at all if you don't want project owners to have access to it.

Reply:
Right --- confusion ensues when we style links link buttons. I do want them to know it's possible to do this and why they can't.

Reply:
Wrap the text in a span or div that's styled to look like a disabled button? One massive bug in the project builder at the moment is that you can tab to, and activate, inputs and links that are styled as disabled but not actually disabled in the DOM.